simple/coupler_main.F90

```diff
@@ -53,51 +53,16 @@ program coupler_main
                               atmos_ice_boundary_type
                              !land_ice_boundary_type
 
-!--- FMS modules
-use constants_mod,      only: constants_init
-
-use data_override_mod,  only: data_override_init
-
-use diag_manager_mod,   only: diag_manager_init, diag_manager_end, &
-                              get_base_date, diag_manager_set_time_end
-
-use field_manager_mod,  only: MODEL_ATMOS, MODEL_LAND, MODEL_ICE
-
 use flux_exchange_mod,  only: flux_exchange_init,   &
                               sfc_boundary_layer,   &
                               flux_down_from_atmos, &
-                              flux_up_to_atmos,     &
-                              flux_exchange_end       ! may not be used?
-
-use fms_affinity_mod,   only: fms_affinity_init, fms_affinity_set
-
-use fms_mod,            only: check_nml_error,  &
-                              error_mesg, fms_init, fms_end,         &
-                              write_version_number, uppercase, stdout
-
-use fms_io_mod,         only: fms_io_exit !< This can't be removed until fms_io is not used at all
-
-use fms2_io_mod,        only: file_exists, ascii_read
-
-use mpp_mod,            only: mpp_init, mpp_pe, mpp_root_pe, mpp_npes, mpp_get_current_pelist, &
-                              stdlog, mpp_error, NOTE, FATAL, WARNING, input_nml_file
-use mpp_mod,            only: mpp_clock_id, mpp_clock_begin, mpp_clock_end
-
-use mpp_mod,            only: mpp_chksum, mpp_set_current_pelist
-
-use mpp_domains_mod,    only: mpp_get_global_domain, mpp_global_field, CORNER, mpp_set_domain_symmetry
-
-use time_manager_mod,   only: time_type, set_calendar_type, set_time,  &
-                              set_date, days_in_month, month_name,     &
-                              operator(+), operator (<), operator (>), &
-                              operator (/=), operator (/), get_date,   &
-                              operator (*), THIRTY_DAY_MONTHS, JULIAN, &
-                              GREGORIAN, NOLEAP, NO_CALENDAR
-
-use tracer_manager_mod, only: tracer_manager_init, get_tracer_index, &
-                              get_number_tracers, get_tracer_names,  &
-                              NO_TRACER
+                              flux_up_to_atmos!,     &
+                              !flux_exchange_end       ! may not be used?
+!--- FMS modules
+use FMS, version_fms=>version
```

Comment:
Not sure we want to publicize version from FMS simply as 'version'.  It probably should be exported as version_fms from the beginning.